recaptcha_enterprise/demosite/app/package.json

```diff
@@ -1,19 +1,20 @@
 {
-  "name": "recaptcha-demosite",
-  "description": "",
-  "version": "1.0.0",
-  "scripts": {
-      "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-      "type": "git",
-      "url": "https://github.com/googleapis/nodejs-recaptcha-enterprise.git"
-  },
-  "license": "MIT",
-  "dependencies": {
-      "@google-cloud/recaptcha-enterprise": "^3.0.0",
-      "body-parser": "^1.20.0",
-      "express": "^4.18.1",
-      "mustache-express": "^1.3.2"
-  }
+    "name": "recaptcha-demosite",
+    "description": "",
+    "version": "1.0.0",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/googleapis/nodejs-recaptcha-enterprise.git"
+    },
+    "license": "MIT",
```

Comment:
Confirming we really want to use MIT license here? Also notice some license headers use Apache, so we'll need to be consistent